src/background/fetch-sse.ts

```diff
@@ -22,3 +22,16 @@ export async function fetchSSE(
     parser.feed(str)
   }
 }
+
+export async function fetchSimple(
+  resource: string,
+  options: RequestInit & { onMessage: (message: string) => void },
+) {
+  const { onMessage, ...fetchOptions } = options
+  const resp = await fetch(resource, fetchOptions)
+  if (!resp.ok) {
+    const error = await resp.json().catch(() => ({}))
+    throw new Error(!isEmpty(error) ? JSON.stringify(error) : `${resp.status} ${resp.statusText}`)
+  }
+  resp.text().then((t) => onMessage(t))
```

Comment:
why not using the streaming option? does not the LLAMA endpoint provide a streaming response? This style would 
 result in long turning-around time. In other words, users should wait for a while before seeing the result. Is this intentional? Or is this for the experimentation like POC?

Reply:
Sure, it supports. I have changed it to streaming way.